apiserver/facades/internal/package_test.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package internal_test
```

Comment:
Prefer to put package_test.go in the non-test package, along with mocks.

Reply:
sure, done